plugins/gatsby-remark-autolink-headers-custom/index.js

```diff
@@ -64,10 +63,15 @@ module.exports = (
     if (icon !== false) {
       patch(data.hProperties, `style`, `position:relative;`);
       const label = id.split(`-`).join(` `);
-      const method = isIconAfterHeader ? `push` : `unshift`;
+      const method = `push`;
       node.children[method]({
```

Comment:
kind of a nit but if the method isn't being chosen dynamically anymore we should just call it
```suggestion
      node.children.push({
```